src/halodrops/processor.py

```diff
@@ -967,9 +986,32 @@ def add_q_and_theta_to_l2_ds(self):
         else:
             ds = self.l2_ds
 
-        ds = calc_q_from_rh(ds)
-        ds = calc_theta_from_T(ds)
+        ds = hh.calc_q_from_rh(ds)
+        ds = hh.calc_theta_from_T(ds)
 
         object.__setattr__(self, "_interim_l3_ds", ds)
 
         return self
+
+
+@dataclass(order=True, frozen=True)
+class Gridded:
+    sondes: dict
+    # platforms: dict = field(default_factory=dict)
+    _: KW_ONLY
+    interpolation_grid: np.ndarray = field(
```

Comment:
These class attributes of `interpolation_grid`, `interpolation_bins` and `max_gap_fill` will make it fixed values, and unable to be changed by user config. I think it is better to have these values as something that can be imported from a `helper` function. The `concat_sondes` can then use these as arguments.